content/en/database_monitoring/connect_dbm_and_apm.md

```diff
@@ -72,6 +72,13 @@ SQL Server and Oracle do not support `full` propagation mode due to statement ca
 
 \* [CommandType.StoredProcedure][25] not supported
 
+\*\* Full mode SQL Server/Java:
+- This mode is in beta.
```

Comment:
```suggestion
```

I don't think this communicates anything. My question as a customer would be "what does beta mean?"

Reply:
Here `beta` means: "This is a brand new feature. We don't have any known issues, but not many customers are using it.". What would be the best way to put it? Shall we omit `beta` completely?

Reply:
I would remove this line

Reply:
removed now